app/decorators/ext_management_system_decorator.rb

```diff
@@ -6,6 +6,10 @@ def fonticon
   end
 
   def listicon_image
-    "100/vendor-#{image_name}.png"
+    if kind_of?(ManageIQ::Providers::Hawkular::MiddlewareManager)
+      'vendor-hawkular'
+    else
+      "100/vendor-#{image_name}.png"
+    end
```

Comment:
@pilhuhn We're switching from PNGs to SVGs for all vendor icons. Please use:
"svg/vendor-#{image_name}.svg"